R/nm-file.R

```diff
@@ -96,7 +96,9 @@ nm_data <- function(.mod) {
   check_model_object(.mod, c(NM_MOD_CLASS, NM_SUM_CLASS))
   .path <- get_data_path(.mod)
   verbose_msg(glue("Reading data file: {basename(.path)}"))
-  .d <- as_tibble(fread(.path, na.strings = ".", verbose = FALSE))
+  .d <- fread(.path, na.strings = ".", verbose = FALSE)
+  colnames(.d) <- make.unique(.d %>% colnames(), sep="_")
+  .d <- as_tibble(.d)
```

Comment:
I recall talking about this. I think we said we might as well put something in this function as well just in case? If so im wondering if we want the same error handling as we have in `nm_file_impl`, in which case we'd want another (similar) test